tests/integration/files/log_handlers/runtests_log_handler.py

```diff
@@ -35,8 +34,6 @@
 def __virtual__():
     if 'runtests_log_port' not in __opts__:
         return False, "'runtests_log_port' not in options"
-    if six.PY3:
```

Comment:
can we revert this for now since we showed it hanging tests on the 2019.2.1 branch?

Reply:
Done.